compiler/lib/eval.ml

```diff
@@ -170,14 +187,18 @@ type is_int =
   | N
   | Unknown
 
-let is_int info x =
+let is_int ~target info x =
   match x with
   | Pv x ->
       get_approx
         info
         (fun x ->
           match Flow.Info.def info x with
           | Some (Constant (Int _)) -> Y
+          | Some (Constant (NativeInt _ | Int32 _)) -> (
```

Comment:
I though it NativeInt and Int32 were not used with the js backend ..

Reply:
Well spotted. I replaced these with assertions that the backend is Wasm instead.